source/Meadow-Filter-Parse.js

```diff
@@ -45,6 +45,37 @@
 
  This means: FBV~Category~EQ~Books~FBV~PublishedYear~GT~2000~FSF~PublishedYear~DESC~0
              Filters down to ALL BOOKS PUBLUSHED AFTER 2000 IN DESCENDING ORDER
+
+ Similar queries are supported for fields that contain a JSON object The FIELD is in a dot notation.
+ Basic Anatomy of FIELD:
+        TABLECOLUMN.KEY
+		where TABLECOLUMN is the record column according to the schema
+		and KEY is a property of the JSON contained in TABLECOLUMN. `.KEY` may be repeated for nested JSON objects.
+ FBJV - Filter By JSON Value (left-side AND connected)
+       FBJV~FormData.Meta.FeatureEnabled~EQ~True
+       Possible comparisons:
+       * EQ - Equals To (=)
+       * NE - Not Equals To (!=)
+       * GT - Greater Than (>)
+       * GE - Greater Than or Equals To (>=)
+       * LT - Less Than (<)
+       * LE - Less Than or Equals To (<=)
+       * LK - Like (Like)
+       * IN - Is NULL
+       * NN - Is NOT NULL
+       * INN - IN list
+       * NIN - NOT IN list
+ FBJVOR - Filter By JSON Value (left-side OR connected)
+ FBJL - Filter By JSON List (value list, separated by commas)
+       FBJL~FormDataJSON.Tidings.Data.Type~INN~ChangeOrder,Payment Form
+ FBJLOR - Filter By JSON List (value list, separated by commas, left-side OR connected)
+       FBJLOR~FormDataJSON.Tidings.Data.Type~INN~ChangeOrder,Payment Form
+ FBJD - Filter by JSON Date (exclude time)
+       FBJD~FormDataJSON.UpdateDate~EQ~2015-10-01
+ FSJF - Filter Sort JSON Field
+       FSJF~FormDataJSON.Category~ASC~0
+       FSJF~FormDataJSON.Category~DESC~<cast>
+          where <cast> can be in (UINT,SINT,CHAR,DD,DATE) - defaults to CHAR
```

Comment:
We should have a legend somewhere in this documentation for these types.

If someone uses this, what ends up happening if a value in a record doesn't match the requested type?

Reply:
MySQL returns a default value for that type. Example with JCCard JCSequenceNumber, where some sequence numbers are actually alphanumeric:
`JCSeqNumCastToDecimal`
0.00000
-----
`Original/Raw JCSequence Number`
JC 621003`